airflow/providers/microsoft/azure/transfers/local_to_adls.py

```diff
@@ -96,3 +96,53 @@ def execute(self, context: Context) -> None:
             blocksize=self.blocksize,
             **self.extra_upload_options,
         )
+
+
+class DataToADLSOperator(BaseOperator):
```

Comment:
I don't understand how the azure changes are related to the `executemany` added in sql provider?

Reply:
I will close this one I think I started by mistake on same branch from other pull request that is still open